tests/factories/cht/generate.js

```diff
@@ -51,7 +60,63 @@ const createData = ({ healthCenter, user, nbrClinics=10, nbrPersons=10 }) => {
   return { clinics, reports, persons };
 };
 
-const createHierarchy = ({ name, user=false, nbrClinics=50, nbrPersons=10 }) => {
+const createDataWithRealNames = ({ healthCenter, user, nbrClinics = 10, nbrPersons = 10 }) => {
+  const clinics = Array.from({ length: nbrClinics }).map((_, index) => {
+    const firstName = firstNames[index % firstNames.length];
+    const lastName = lastNames[index % lastNames.length];
+    const personName = `${firstName} ${lastName}`;
+    const personPhoneNumber = phoneNumbers[index % phoneNumbers.length];
+
+    const primaryContact = personFactory.build({
+      name: personName,
+      phone: personPhoneNumber
+    });
+
+    const clinic = placeFactory.place().build({
+      type: 'clinic',
+      parent: { _id: healthCenter._id, parent: healthCenter.parent },
+      name: `${personName} Family`,
+      contact: primaryContact
+    });
+
+    primaryContact.parent = { _id: clinic._id, parent: clinic.parent };
+
+    const additionalPersons = Array.from({ length: nbrPersons - 1 }).map((_, i) => {
+      const additionalPersonName = `${firstNames[i % firstNames.length]} ${lastNames[i % lastNames.length]}`;
+      const additionalPhoneNumber = phoneNumbers[i % phoneNumbers.length];
+      return personFactory.build({
+        parent: { _id: clinic._id, parent: clinic.parent },
+        name: additionalPersonName,
+        patient_id: patientIds[i % patientIds.length],
+        phone: additionalPhoneNumber
+      });
+    });
+
```

Comment:
When chaining functions, each function goes in a new line
```suggestion
const additionalPersons = Array
  .from({ length: nbrPersons - 1 })
  .map((_, i) => {
      const additionalPersonName = `${firstNames[i % firstNames.length]} ${lastNames[i % lastNames.length]}`;
      const additionalPhoneNumber = phoneNumbers[i % phoneNumbers.length];
      return personFactory.build({
        parent: { _id: clinic._id, parent: clinic.parent },
        name: additionalPersonName,
        patient_id: patientIds[i % patientIds.length],
        phone: additionalPhoneNumber
      });
  });

```